### DISCUSSION TITLE
Would like to see the list of penalized rounds for each account

### DISCUSSION THREAD

**Discussion by Cryolite** [smly/mjai.app#15] [Ideas]
I would like to see the list of rounds in which bots submitted by an account was penalized (the bots returned an invalid response to the game engine), and download the corresponding server logs.

Currently, in order to obtain the list of such rounds, it is necessary to go through all the rounds in which the bots has participated, which is very time-consuming. I also don't know how to download the server log for each round.

Since investigating the server log of the rounds in which bots are penalized is very important for debugging the bots, I would appreciate it if you would consider this request.

**Comment by smly** [comment]
Totally agreed with you. I will do it. Unfortunately, right now all server logs and bot I/O logs are not stored. Only server errors and mjai logs are stored. I will save them and provide them to users for debugging soon.
